crates/bevy_color/Cargo.toml

```diff
@@ -23,7 +23,7 @@ derive_more = { version = "1", default-features = false, features = [
   "from",
   "display",
 ] }
-wgpu-types = { version = "22", default-features = false, optional = true }
+wgpu-types = { git = "https://github.com/gfx-rs/wgpu", branch = "trunk", default-features = false, optional = true }
```

Comment:
It seems dangerous to move to a trunk branch no? Shouldn't we specify a locked commit hash instead? (even if it's the same as v23 at the time of writing)

Reply:
wgpu 0.23 isn't released yet, presumably the author intends to update this and take the PR out of draft status once it is released.